spec/index_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe 'Index Controller' do
+
+  it 'Should get the / route' do
+    get '/'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('Sealions')
+  end
+
+  #CREATE
+  it 'Should process a post request' do
+    expect{
+      post '/sealions', {name: 'Lauren'}
+    }.to change(Sealion, :count).by(1)
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('Sealions')
+  end
+
+  #READ
+  it 'Should process a read request on id 3' do
+    get '/sealions/3'
```

Comment:
Hard coding the Sealion's id makes your test relay on the data in your database, we can always assume that data will be the same. Let's try making  let block to define a sealion and using that id here